### PR TITLE
added axios

### PR DESCRIPTION
**Pull request by himanshujaidka**
## Related Issuse

- Info about Issue or bug

Closes: #207 [issue number that will be closed through this PR]

#### Describe the changes you've made
I have added Axios library and created a file respectively
A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [x] I have made corresponding changes to the documentation.(Specially for Backend)

### CODE REVIEW

**Comment by himanshujaidka** [comment]
@codewithvk @purnima143, pls review

**Review by codewithvk — CHANGES_REQUESTED** [review]
@himanshujaidka Please remove refactor component commits.

File: client/src/urlConfig.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export const api = "http://localhost:5000/api";
```
Comment (codewithvk) [inline client/src/urlConfig.js:1]: declare in ENV file and update env.example

File: client/src/helpers/axios.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+});
+
+axiosIntance.interceptors.request.use((req) => {
+    return req;
+})
+axiosIntance.interceptors.response.use((res) => {
+    return res;
+}, (error) => {
+    console.log(error.response);
+    // const { status } = error.response;
```
Comment (codewithvk) [inline client/src/helpers/axios.js:22]: remove comments

File: client/src/urlConfig.js (hunk 1 above)
Comment (himanshujaidka) [reply client/src/urlConfig.js:1]: ok

File: client/src/helpers/axios.js (hunk 2 above)
Comment (himanshujaidka) [reply client/src/helpers/axios.js:22]: ok

**Comment by himanshujaidka** [comment]
@codewithvk pls review now

**Pull request merged**